### PR TITLE
v1.3 Changes

### PR DESCRIPTION
**Pull request by darkslyde**
Request merge branch into master for release.

### CODE REVIEW

File: extension.xml [hunk 1]
```diff
@@ -3,23 +3,27 @@
...
+    <!-- Please see the license.md file included with this distribution for attribution and copyright information. -->
+
+    <!-- Why in Lathander's name are you trying to load order this at 2000?! Because hook reasons-->
   	<loadorder>2000</loadorder>
     <author>matjam</author>
     <description>Show the status of a PC or NPC on the battlemap using graphical icon overlays.</description>
   </properties>
-
-  <announcement text="5E - Matjam's Save Indicators version 1.0" font="emotefont" icon="MatjamsExtension" />
+  <!-- Changed name from Save to Status on version 1.3 dated 7/2/20 -->
```
Comment (matjam) [inline extension.xml]: doesn't need to be commented here; have the commit history and git blame if I need to figure it out.

Comment (matjam) [inline extension.xml]:
```suggestion
```

you can commit this suggestion to remove the line

File: extension.xml [hunk 2]
```diff
@@ -3,23 +3,27 @@
...
+  <announcement text="5E - Matjam's Status Indicators version 1.3 \rUpdates can be found at: https://github.com/matjam/5e-status-indicators/releases" font="emotefont" icon="MatjamsExtension" />
```
Comment (matjam) [inline extension.xml:12]: good catch

**Review by matjam — CHANGES_REQUESTED** [review]

File: extension.xml (hunk 1 above)
Comment (darkslyde) [reply extension.xml]: Is my Persuasion Check roll of 19 not enough?! Fine!

**Review by matjam — DISMISSED** [review]

**Review by darkslyde — COMMENTED** [review]
Please merge after final approval

**Review by matjam — APPROVED** [review]

**Pull request merged**